cmake/ExternalMFEM.cmake

```diff
@@ -231,8 +249,9 @@ message(STATUS "MFEM_OPTIONS: ${MFEM_OPTIONS_PRINT}")
 set(MFEM_PATCH_FILES
   "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_mesh_part.diff"
   "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_mesh_vis.diff"
-  "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_submesh.diff"
   "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_direct_solvers.diff"
+  "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_pa_prereq.diff"
+  "${CMAKE_SOURCE_DIR}/extern/patch/mfem/patch_pa_libceed.diff"
```

Comment:
Just for the record, which branches in mfem are these patches based on? Also, the submesh patch is gone because it's now on main?

Reply:
Yep ND and RT `SubMesh` support is in MFEM `master`.

Here the `pa_prereq` patch is based on `sjg/assembly-cleanup-prereqs` (diff from `master`) and `pa_libceed` is from there to `sjg/libceed-openmp-dev`.